pkg/images/images.go

```diff
@@ -24,7 +24,7 @@ var (
 )
 
 const (
-	ValidDseVersionRegexp      = "6\\.8\\.\\d+"
+	ValidDseVersionRegexp      = "(6\\.8\\.\\d+)|(7\\.\\d+\\.\\d+)"
 	ValidOssVersionRegexp      = "(3\\.11\\.\\d+)|(4\\.\\d+\\.\\d+)"
```

Comment:
And add 5.x.x stuff here as well
```suggestion
	ValidOssVersionRegexp      = "(3\\.11\\.\\d+)|(4\\.\\d+\\.\\d+)|(5\\.\\d+\\.\\d+)"
```